charts/multicluster-scheduler/templates/cm.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels: {{ include "labels" . | nindent 4 }}
 data:
   proxy-scheduler-config: |
-    apiVersion: kubescheduler.config.k8s.io/v1beta3
+    apiVersion: kubescheduler.config.k8s.io/v1
```

Comment:
AFAIU, this is necessary to support K8s 1.29 and above. However, it means dropping support for K8s 1.24, which is fair.

Reply:
yup, I think its reasonable since it reached EOL almost a year ago.